.gitignore

```diff
@@ -20,3 +20,7 @@ coverage
 .netlify/plugins
 
 functions/*.zip
+
+# schemas
+
+gbfs-validator/versions/schemas/.*
```

Comment:
question: Why do we need to ignore files under this folder and have files on it already check-in in the repo?

Reply:
We just ignore "dot files" (and repositories) from the imported repo https://github.com/MobilityData/gbfs-json-schema , because they are not needed/relevant here. Related to that other comment: https://github.com/MobilityData/gbfs-validator/pull/104#pullrequestreview-1491739949

Reply:
If these files are not are not needed/relevant here, can we remove them from the repo instead of ignoring them in the .gitignore?

Reply:
They were removed with `git rm`. The gitignore is here to prevent similar files to be imported again from the other repository.

Reply:
Perfect, thank you!